ValveResourceFormat/IO/FileExtract.cs

```diff
@@ -63,11 +64,67 @@ public static ContentFile Extract(Resource resource)
 
                 case ResourceType.Texture:
                     {
-                        using var bitmap = ((Texture)resource.DataBlock).GenerateBitmap();
+                        var bitmap = ((Texture)resource.DataBlock).GenerateBitmap();
```

Comment:
So we need to keep it around for ImageExtract? That sounds like it never gets explicitly exposed.

Reply:
Yup it's left up to the GC.